docs/BENCH-CONFIG-SPEC.md

```diff
@@ -62,6 +64,17 @@ Formatting example:
 | `algorithm`:`sklearnex_context` | None |  | Parameters for sklearnex's config_context used over estimator. Updated by `sklearn_context` if set. |
 | `bench`:`ensure_sklearnex_patching` | True |  | If True, warns about sklearnex patching failures. |
 
+### `distr_sklearn_estimator`
+
+| Parameter keys | Default value | Choices | Description |
+|:---------------|:--------------|:--------|:------------|
+| `algorithm`:`estimator` | None |  | Name of measured estimator. |
+| `algorithm`:`estimator_params` | Empty `dict` |  | Parameters for estimator constructor. |
+| `algorithm`:`num_procs` | None | Any integer less than or equal to number of GPUs available | Number of GPUs to use in distributed benchmarking. |
```

Comment:
generally num_procs can be higher than num_gpus or num-cpus to be used. But for the purpose of benchmarking, we would like to use same num_procs as num_gpus. Also num_procs refers to number of processes to be created by MPI, which then are pinned to individual gpus or cpus.